warehouse/forklift/legacy.py

```diff
@@ -487,6 +487,7 @@ def _sort_releases(request: Request, project: Project):
     require_csrf=False,
     require_methods=["POST"],
     has_translations=True,
+    permit_duplicate_post_keys=True,
```

Comment:
As far as I'm aware, this is the only view where we actually depend on and permit duplicate keys.